### PR TITLE
Return empty datastream when cocina exists in postgres.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
This will allow testing cocina-only objects (i.e., not saved to Fedora) without having to disable datastream editing in Argo. (It just won't be supported for cocina-only objects since Argo will receive an empty datastream list for those.)

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

Unit

### CODE REVIEW

File: app/controllers/datastreams_controller.rb [hunk 1]
```diff
@@ -7,10 +7,14 @@ class DatastreamsController < ApplicationController
   before_action :load_item
 
   def index
-    result = @item.datastreams
-                  .reject { |name, instance| instance.new? || name == 'workflows' }
-                  .values
-                  .map { |ds| serialize_datastream(ds) }
+    result = if Settings.enabled_features.postgres.ar_find && CocinaObjectStore.new.ar_exists?(@item.pid)
```
Comment (jcoyne) [inline app/controllers/datastreams_controller.rb:10]: Is there a reason we show the data streams if it isn't in PG?

Comment (justinlittman) [reply app/controllers/datastreams_controller.rb:10]: It preserves the ability to edit the datastreams for objects that are stored in Fedora (e.g., for remediation).

Comment (jcoyne) [reply app/controllers/datastreams_controller.rb:10]: Would you mind adding this as a comment here so that we don't forget the reason for this choice?

Comment (justinlittman) [reply app/controllers/datastreams_controller.rb:10]: I don't mind. Done.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**